libraries/core-react/src/components/DatePicker/DatePicker.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Meta, Story } from '@storybook/react/types-6-0'
+import { DatePicker, DatePickerProps } from './DatePicker'
+import { TimePicker } from '../TimePicker/TimePicker'
+
+const Container = styled.div`
+  height: 800px;
+  width: 100%;
+  padding: 32px;
+  box-sizing: border-box;
+  display: grid;
+  grid-template-rows: 200px;
+  grid-gap: 32px 32px;
+  background: #ebebeb;
+`
+
+const DatePickerWrapper: React.FC<DatePickerProps> = ({
```

Comment:
This wrapper breaks the Storybook props table (not showing all properties) and "show code".